web/yo/app/scripts/directives/driverealtimestring.js

```diff
@@ -64,7 +64,7 @@ angular.module('oncokbApp')
                         if(scope.rs && _.isNull(scope.rs.get('lastReviewed')) && scope.rs.get('action') !== 'rejected') {
                             scope.rs.set('lastReviewed', o);
                         }
-                        scope.content.stringO = stringUtils.getTextString(scope.object.text);
+                        scope.content.stringO = scope.object.text;
```

Comment:
Nope. The cause for the text style issue is I used angualr expression instead of ng-model.